script.js

```diff
@@ -0,0 +1,62 @@
+class Books {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+const buttonAdd = document.querySelector('.button-add');
+const displayBooks = document.getElementById('books-list');
+const bookShelf = [];
+
+// Storage functions
+
+function storeGet() {
+  let books;
+  if (localStorage.getItem('books') === null) {
+    books = [];
+  } else {
+    books = JSON.parse(localStorage.getItem('books'));
+  }
+
+  return books;
+}
+function storeAdd(newBook) {
+  const books = storeGet();
+  books.push(newBook);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+function storeDelete(title) {
+  const books = storeGet();
+  const index = books.findIndex((book) => book.title === title);
+  if (index !== -1) {
+    books.splice(index, 1);
```

Comment:
- [ OPTIONAL ] I would suggest using [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) instead of [splice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/splice) because the first one does not mutate the array